[Bug]: can't install/update and start server

### User story

Error occurs during installation/update/start

### Game

Vintagestory

### Linux distro

Other

### Command

command: update, command: install, command: start

### Install/Update

#### Further information

The developers have changed the json structure in the latest updates. Fix: replace 'server' to 'linuxserver' in update_vintagestory.sh

#### Relevant log output

```shell
fetching  null...curl: (6) Could not resolve host: null
```

#### Steps to reproduce

./vintsserver update or ./vintsserver install

### Start after install/update fix

#### Further information

Fix: Now the server requires dotnet 7 runtime, and changing some vars:

```shell
startparameters="--dataPath ${servercfgdir}"
executable="./VintagestoryServer"
preexecutable=""
```

#### Relevant log output

```shell
Unhandled Exception:
System.TypeInitializationException: The type initializer for 'Vintagestory.API.Config.RuntimeEnv' threw an exception. ---> System.TypeLoadException: Could not resolve type with token 010000d6 from typeref (expected class 'System.AppDomain' in assembly 'System.Runtime, Version=7.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a')
   --- End of inner exception stack trace ---
  at (wrapper managed-to-native) System.Object.__icall_wrapper_mono_generic_class_init(intptr)
  at Vintagestory.Server.ServerProgram.Main (System.String[] args) [0x00078] in <9f274c476aca4155adbb348bb301b88b>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.TypeInitializationException: The type initializer for 'Vintagestory.API.Config.RuntimeEnv' threw an exception. ---> System.TypeLoadException: Could not resolve type with token 010000d6 from typeref (expected class 'System.AppDomain' in assembly 'System.Runtime, Version=7.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a')
   --- End of inner exception stack trace ---
  at (wrapper managed-to-native) System.Object.__icall_wrapper_mono_generic_class_init(intptr)
  at Vintagestory.Server.ServerProgram.Main (System.String[] args) [0x00078] in <9f274c476aca4155adbb348bb301b88b>:0
```

#### Steps to reproduce

./vintsserver start

--- a/.github/workflows/labeler.yml
+++ b/.github/workflows/labeler.yml
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Issue Labeler
-        uses: github/issue-labeler@v3.1
+        uses: github/issue-labeler@v3.2
         with:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
           configuration-path: .github/labeler.yml

--- a/lgsm/config-default/config-lgsm/vintsserver/_default.cfg
+++ b/lgsm/config-default/config-lgsm/vintsserver/_default.cfg
@@ -9,7 +9,7 @@
 #### Game Server Settings ####
 
 ## Server Parameters | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
-startparameters="--datapath ${servercfgdir}"
+startparameters="--dataPath ${servercfgdir}"
 
 ## Release Settings | https://docs.linuxgsm.com/game-servers/vintagestory#release-settings
 # Branch (stable|unstable)
@@ -150,8 +150,8 @@ glibc="2.17"
 ## Game Server Directories
 systemdir="${serverfiles}"
 executabledir="${systemdir}"
-preexecutable="mono"
-executable="./VintagestoryServer.exe"
+preexecutable=""
+executable="./VintagestoryServer"
 servercfgdir="${serverfiles}/data/${selfname}"
 servercfg="serverconfig.json"
 servercfgdefault="serverconfig.json"

--- a/lgsm/modules/update_vintagestory.sh
+++ b/lgsm/modules/update_vintagestory.sh
@@ -42,9 +42,9 @@ fn_update_remotebuild() {
 	else
 		remotebuildversion=$(echo "${remotebuildresponse}" | jq -r '[ to_entries[] ] | .[].key' | grep -E "\-rc|\-pre" | sort -r -V | head -1)
 	fi
-	remotebuildfilename=$(echo "${remotebuildresponse}" | jq --arg remotebuildversion "${remotebuildversion}" -r '.[$remotebuildversion].server.filename')
-	remotebuildurl=$(echo "${remotebuildresponse}" | jq --arg remotebuildversion "${remotebuildversion}" -r '.[$remotebuildversion].server.urls.cdn')
-	remotebuildhash=$(echo "${remotebuildresponse}" | jq --arg remotebuildversion "${remotebuildversion}" -r '.[$remotebuildversion].server.md5')
+	remotebuildfilename=$(echo "${remotebuildresponse}" | jq --arg remotebuildversion "${remotebuildversion}" -r '.[$remotebuildversion].linuxserver.filename')
+	remotebuildurl=$(echo "${remotebuildresponse}" | jq --arg remotebuildversion "${remotebuildversion}" -r '.[$remotebuildversion].linuxserver.urls.cdn')
+	remotebuildhash=$(echo "${remotebuildresponse}" | jq --arg remotebuildversion "${remotebuildversion}" -r '.[$remotebuildversion].linuxserver.md5')
 
 	if [ "${firstcommandname}" != "INSTALL" ]; then
 		fn_print_dots "Checking remote build: ${remotelocation}"

